### PR TITLE
Disallow bulkio statements within transactions

### LINKED ISSUE #2056: Clarify and restrict the txn semantics of statements for bulk I/O
Background: https://airtable.com/tblD3oZPLJgGhCmch/viwINAgXA5on9pr2T/recNBM4ZwFJxf258L

PRs: [sqlccl: disallow bulk IO statements in txn](https://github.com/cockroachdb/cockroach/pull/21488)

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #2056

### CODE REVIEW

File: v2.0/backup.md [hunk 1]
```diff
@@ -149,7 +151,7 @@ AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00';
 
 ~~~ sql
 > BACKUP bank.customers, bank.accounts TO 'azure://acme-co-backup/tables-accounts-customers-2017-03-27-full?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co'
-AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00');
+AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00';
```
Comment (jseldess) [inline v2.0/backup.md]: Aren't you updating these in another PR? If so, you might want to revert this change so as to not create a conflict for yourself.

Comment (Amruta-Ranade) [reply v2.0/backup.md]: Hmm..good point. Thanks!
